### PR TITLE
Move the sigmoid activation to the model itself

### PR DESCRIPTION
**Pull request by rht**
This is part of the effort to migrate `train.py` to using Ignite [1].

[1] https://github.com/pytorch/ignite

### CODE REVIEW

**Comment by milesial** [comment]
Thanks ! That is indeed cleaner as most of the time we want the probabilities and not the raw logits.
This project is now too old and I don't feel like migrating it to ignite, but feel free to open some PR if you wish !
